README.md

```diff
@@ -135,6 +135,9 @@ kube-system                kubernetes-dashboard-5f7b999d65-ntrxb         1/1
 
 You should review files `cluster.yaml` and `machines.yaml`, and ensure the `domain`, `dataCenter` and `osReferenceCode` are set. You can customize those values based on your requirement.
 
+For `machines.yaml`, you can follow the [sample file](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/master/examples/ubuntu/machines.yaml), and make sure `sshKeyName` and the label in `ibmcloud` are identical.
+
```

Comment:
kill this line.

Also please ask @songleo for how to squash your commit to one commit.

Reply:
I don't fully understand 
`sshKeyName` and the label in `ibmcloud` are identical.

can you help to provide detail?

Reply:
The `sshKeyName` in `machines.yaml` should be identical to the `label` value in Softlayer SSH KEY

![image](https://user-images.githubusercontent.com/24581945/59398690-ac473e00-8dc3-11e9-8a64-e4a7e84a95fa.png)

![image](https://user-images.githubusercontent.com/24581945/59398732-d26cde00-8dc3-11e9-990b-1daba2e802a4.png)

Reply:
you may add those pictures, for reader to understand what's the meaning here
and please note actually you can have more ssh keys created in cloud and you will select one of them in the machines.yaml

Reply:
I will add those pictures. Thanks for your advice.